### DISCUSSION TITLE
How to interpret the pseudo count sensitivity analysis

### DISCUSSION THREAD

**Discussion by janettatop** [FrederickHuangLin/ANCOMBC#172] [Q&A] [answered]
When I check the output of the pseudo count sensitivity analysis I observed that not all taxa with a score of "0" were considered significant. Could you explain why?

In addition, there were also a high number of taxa with a sensitivity score close to "0".  Would it be valid to consider a cut-of and also consider taxa as significant with a sensitivity score between e.g. 0 and 0.01?

**Comment by FrederickHuangLin** [comment, marked as answer]
Hi @janettatop,

Thank you for your question! We appreciate your feedback regarding the sensitivity analysis in the previous version. We have taken your concerns into consideration and made significant improvements to the sensitivity analysis in the latest version (2.3.1) of the package.

In the updated version, the sensitivity analysis now involves adding a range of pseudo-counts (ranging from 0.01 to 0.5 in increments of 0.01) to the zero counts of each taxon. Linear regression models are then performed on the bias-corrected log abundance table using these different pseudo-counts. The sensitivity score for each taxon is calculated as the proportion of times the p-value exceeds the specified significance level (alpha). If all p-values consistently show significance or nonsignificance across the different pseudo-counts and are consistent with the results obtained without adding pseudo-counts to zero counts (using the default settings), then the taxon is considered not sensitive to the pseudo-count addition.

You can find the updated package (version 2.3.1) on this GitHub Repository. We have also submitted a revised version of the ANCOM-BC2 paper, and we hope that the preprint of the paper will be available in approximately 2 weeks.

Best regards,
Huang
